docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc

```diff
@@ -0,0 +1,66 @@
+[id='glusterfs-reconfig-proc']
+= Changing GlusterFS configuration
+
+You must check whether your OpenShift environment uses GlusterFS to provide permanent storage volumes. If it uses GlusterFS, you must change the GlusterFS configuration.
+
+.Procedure
+
+. Run the following command:
++
+[subs="verbatim,macros"]
+----
+oc get storageclass 
+----
++
+In the results, check whether the `(default)` marker is on the storage class that lists `glusterfs`. For example, in the following output the default storage class is `gluster-container`, which does list `glusterfs`:
++
+[subs="verbatim,macros"]
+----
+NAME              PROVISIONER                       AGE
+gluster-block     gluster.org/glusterblock          8d
+gluster-container (default) kubernetes.io/glusterfs 8d
+----
++
+If the result has a default storage class that does not list `glusterfs` or if the result is empty, you do not need to make any changes. In this case, skip the rest of this procedure.
+
+. Run the following command to save the configuration of the default storage class into a YAML file:
++
+[subs="verbatim,macros"]
+----
+oc get storageclass <class_name> -o yaml >storage_config.yaml
+----
++
+Where `class-name` is the name of the default storage class. For example:
++
+[subs="verbatim,macros"]
+----
+oc get storageclass gluster-container -o yaml >storage_config.yaml
+----
++
+. Edit the `storage_config.yaml` file:
+.. Remove the lines with the following keys:
+** `creationTimestamp`
+** `resourceVersion`
+** `selfLink`
+** `uid`
+.. On the line with the `volumeoptions` key, add the following two options: `features.cache-invalidation on, performance.nl-cache on`
```

Comment:
I would show them what the line would then look like, for example:

`volumeoptions: client.ssl off, server.ssl off, features.cache-invalidation on, performance.nl-cache`